### ISSUE TITLE
commentFormatting not check // nolint comments

### ISSUE THREAD

**Issue by Bysmyyr** [go-critic/go-critic#1050]
Hi, thanks for create linter!

commentFormatting check do not check nolint comments, for example `//nolint:gocognit` is fine by it(missing space after //).

**Comment by cristaloleg** [comment]
Hi, maybe I misunderstood you, but you're saying that go-critic should report such comment?

```
// nolint:gocognit
func foo() {}
```

?

**Comment by Bysmyyr** [comment]
Hi, thanks for your reply,

It should report this
```
//nolint:gocognit
func foo() {}
```
and accepts this
```
//nolint:gocognit
func foo() {}
```

So just don't have any special rules for these nolint comments

**Comment by cristaloleg** [comment]
Hm...I think that's not a case for `commentFormatting`. `//nolint:foo` is a special comment across linters, probably you're looking for something like https://github.com/ashanbrown/nolintlint or maybe we need a new linter (param for `commentFormatting`?) which reports `//nolint` comments.

Would like to hear @quasilyte opinion (probably he's afk for some time, but pinging)

**Comment by Bysmyyr** [comment]
nolintlint checks that there is max 1 space but 0 and 1 spaces got passed. I ask from them too.

**Comment by Bysmyyr** [comment]
Nolintlint can be confiqured to remove spaces so it saves my day, thanks for hints!
